### DISCUSSION TITLE
Some themes fail to update

### DISCUSSION THREAD

**Discussion by torn4dom4n** [LekoArts/gatsby-themes#924] [Questions] [answered]
## Description

Since v3.0.8, It fails to update some themes in this project. I use GitHub actions and you can see this for [detail](https://github.com/AREA44/gatsby-starter-emilia/runs/6726366029?check_suite_focus=true). I think this error happens because `gatsby-plugin-mdx` package.

**Comment by LekoArts** [comment]
Hi!

I can't reproduce this, both locally and in Gatsby Cloud (for https://github.com/LekoArts/gatsby-starter-portfolio-emilia) it builds successfully. Maybe your GitHub action needs adjustment? Try `npm install --legacy-peer-deps`

**Reply by torn4dom4n** [reply]
I tried `npm install --legacy-peer-deps` but it don't have any thing changes. I don't use Gatsby Cloud, instead of I use GitHub Action and Vercel (https://github.com/torn4dom4n/torn4dom4n.github.io/pull/73). BTW, I tried to use Codespaces of GitHub and run `npm` but it still show this error. Could you fork [my repo](https://github.com/torn4dom4n/torn4dom4n.github.io) and use GitHub Actions or Vercel to reproduce it?

**Comment by torn4dom4n** [comment]
This is list of theme I can't update:
https://github.com/AREA44/gatsby-starter-emilia/pull/8
https://github.com/torn4dom4n/torn4dom4n.github.io/pull/73

**Comment by torn4dom4n** [comment, marked as answer]
I think this error is because of `package-lock.json`. Removed it and everything work fine.
